src/build.d

```diff
@@ -431,6 +432,45 @@ alias checkwhitespace = makeDep!((builder, dep) => builder
     })
 );
 
+alias style = makeDep!((builder, dep)
+{
+    const dir = env["G"].buildPath("dscanner");
```

Comment:
How about `scannerDir`?  `dir` seems a bit to generic

Reply:
Done